src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -6451,7 +6460,8 @@ private enum ParseTypeMode
         {
             Normal,
             Parameter,
-            AfterIsOrCase,
+            AfterIs,
+            AfterCase,
```

Comment:
I see, we do now. Used to pass an extra bool.

---
In reply to: [98789489](https://github.com/dotnet/roslyn/pull/16834#discussion_r98789489) [](ancestors = 98789489)